google-cloud-bigtable-stats/src/test/java/com/google/cloud/bigtable/stats/StatsRecorderWrapperTest.java

```diff
@@ -0,0 +1,511 @@
+                INSTANCE_ID,
+                APP_PROFILE_ID))
+        .isEqualTo(throttlingLatency);
+    assertThat(
+            getAggregationValueAsLong(
+                BuiltinViewConstants.FIRST_RESPONSE_LATENCIES_VIEW,
+                ImmutableMap.of(
+                    BuiltinMeasureConstants.METHOD,
+                    "Bigtable.MutateRow",
+                    BuiltinMeasureConstants.TABLE,
+                    TABLE_ID,
+                    BuiltinMeasureConstants.ZONE,
+                    ZONE,
+                    BuiltinMeasureConstants.CLUSTER,
+                    CLUSTER,
+                    BuiltinMeasureConstants.STATUS,
+                    "UNAVAILABLE",
+                    BuiltinMeasureConstants.CLIENT_NAME,
+                    "bigtable-java"),
+                PROJECT_ID,
+                INSTANCE_ID,
+                APP_PROFILE_ID))
+        .isEqualTo(firstResponseLatency);
+  }
+
+  long getAggregationValueAsLong(
+      View view,
+      ImmutableMap<TagKey, String> tags,
+      String projectId,
+      String instanceId,
+      String appProfileId) {
+    ViewData viewData = wrapper.getViewManager().getView(view.getName());
+    Map<List<TagValue>, AggregationData> aggregationMap =
+        Objects.requireNonNull(viewData).getAggregationMap();
+
+    List<TagValue> tagValues = new ArrayList<>();
+
+    for (TagKey column : view.getColumns()) {
+      if (BuiltinMeasureConstants.PROJECT_ID == column) {
+        tagValues.add(TagValue.create(projectId));
+      } else if (BuiltinMeasureConstants.INSTANCE_ID == column) {
+        tagValues.add(TagValue.create(instanceId));
+      } else if (BuiltinMeasureConstants.APP_PROFILE == column) {
+        tagValues.add(TagValue.create(appProfileId));
+      } else {
+        tagValues.add(TagValue.create(tags.get(column)));
+      }
+    }
+
+    AggregationData aggregationData = aggregationMap.get(tagValues);
+
+    return aggregationData.match(
+        new io.opencensus.common.Function<AggregationData.SumDataDouble, Long>() {
+          @Override
+          public Long apply(AggregationData.SumDataDouble arg) {
+            return (long) arg.getSum();
+          }
+        },
+        new io.opencensus.common.Function<AggregationData.SumDataLong, Long>() {
+          @Override
+          public Long apply(AggregationData.SumDataLong arg) {
+            return arg.getSum();
+          }
+        },
+        new io.opencensus.common.Function<AggregationData.CountData, Long>() {
+          @Override
+          public Long apply(AggregationData.CountData arg) {
+            return arg.getCount();
+          }
+        },
+        new io.opencensus.common.Function<AggregationData.DistributionData, Long>() {
+          @Override
+          public Long apply(AggregationData.DistributionData arg) {
+            return (long) arg.getMean();
+          }
+        },
+        new io.opencensus.common.Function<AggregationData.LastValueDataDouble, Long>() {
```

Comment:
why the full class name? why not an import?
Better yet, can you just use lambdas here?